.github/templates/CITATION.cff.j2

```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+type: software
+message: >
+  If you cite this software, please use these metadata.
```

Comment:
the specs declare this field as requires, but also specifies a default. what does that mean?

why don't they propose to put that as a comment at the top when it's intended for human consumers?

Reply:
idk, they probably want us to specify a msg we are happy with to be extracted/rendered by potential integrations with other services. whats the default?

Reply:
https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#message

Reply:
ah yes. tbh I find the condition "if you use this software then cite" a little bit bossy and therefore chose a different message.